scripts/commit_checker.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+#
+echo ${TRAVIS_COMMIT_MESSAGE}
+
+for line in {1..15}
+do
+    ${line}_line=`echo ${TRAVIS_COMMIT_MESSAGE} | grep -n $line | wc -m`
+    if [ $line -eq 1 ] && [ $(${line}_line) -gt 80 ] ; then
+        echo "ERROR: ${line}_line is too long in Commit Message. It must be less than 80 chars !"
+        exit 1
+    fi
+    if [ $line -eq 2 ] && [ $(${line}_line) -ne 0 ] ; then
+        echo "ERROR: ${line}_line in Commit Message is not zero length"
+        exit 1
+    fi
+    if [ $(${line}_line) -gt 80 ] ; then
+        echo "ERROR: ${line}_line is too long in Commit Message. It must be less than 80 chars !"
+        exit 1
+    fi
+    echo "${line}_line is $(${line}_line) length."
+done
```

Comment:
It misses the check that there is a subject and the body, doesn't it?

Reply:
fixed by git log --pretty=format:'%B' -1

Reply:
Check fo body or subject in included when we check line number in if conditions above.